trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -683,18 +685,20 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
             # quantities for the dictionary are combined correctly. 
                 comm = _get_comm(())
                 if comm.size > 1:
-                    obs_dict_fields = [column_density,tau_ray,delta_lambda,
+                    obs_dict_fields = [column_density,tau_ray,EW, delta_lambda,
                                        lambda_obs, thermal_b, thermal_width]
                     obs_dict_fields = [comm.mpi_allreduce(field,op="sum") for field in obs_dict_fields]
 
-                 # Update the line_observables_dict with values for this line
-                obs_dict = {"column_density":column_density,
+                # Update the line_observables_dict with values for this line
+                EW = np.sum(1-np.exp(-current_tau_field))*self.bin_width
```

Comment:
The EW is traditionally the actual *decrement* in flux due to the optical depth, whereas here, you're calculating it to be the leftover flux after the optical depth has been accounted for.  You can fix this by removing the `1-` at the beginning and just making this: `np.sum(np.exp(-current_tau_field))*self.bin_width`.

Reply:
Isn't `current_tau_field` (which is taken directly from `EW_deposit`) going to be something that starts out at 0, goes up, and then goes back down to 0? In this case, `np.exp(-current_tau_field)` will be all ones near both edges, and the sum you suggest is unbounded. Am I misunderstanding what `EW_deposit` is depositing?